packages/integrations/remix/dmno.meta.json

```diff
@@ -0,0 +1,20 @@
+{
+  "integrationForPackage": "@remix-run/react",
+  "publicPrefix": "VITE_",
+  "installationCodemods": [
+    {
+      "glob": "vite.config.*",
+      "imports": [ {
+          "moduleName": "@dmno/remix-integration",
+          "importVars": [ "dmnoRemixVitePlugin", "dmnoRemixPreset" ],
+        } ],
+      "updates": [ {
```

Comment:
this does not currently include the additional "remix preset" that needs to be inserted